cfme/tests/v2v/test_post_migrations.py

```diff
@@ -1,191 +1,94 @@
-"""Test contains all post migration usecases"""
+"""Tests to validate post-migrations usecases"""
 import re
 
 import fauxfactory
 import pytest
 
 from cfme.base.login import BaseLoggedInPage
-from cfme.fixtures.provider import rhel7_minimal
+from cfme.cloud.provider.openstack import OpenStackProvider
+from cfme.fixtures.v2v_fixtures import get_migrated_vm
 from cfme.infrastructure.provider.rhevm import RHEVMProvider
 from cfme.infrastructure.provider.virtualcenter import VMwareProvider
 from cfme.markers.env_markers.provider import ONE_PER_TYPE
 from cfme.markers.env_markers.provider import ONE_PER_VERSION
 from cfme.utils.appliance.implementations.ui import navigate_to
-from cfme.utils.appliance.implementations.ui import navigator
-from cfme.utils.log import logger
 from cfme.utils.wait import TimedOutError
-from cfme.utils.wait import wait_for
-
 
 pytestmark = [
     pytest.mark.provider(
-        classes=[RHEVMProvider],
+        classes=[RHEVMProvider, OpenStackProvider],
         selector=ONE_PER_VERSION,
-        required_flags=['v2v'],
+        required_flags=["v2v"],
         scope="module"
     ),
     pytest.mark.provider(
         classes=[VMwareProvider],
         selector=ONE_PER_TYPE,
-        fixture_name='source_provider',
-        required_flags=['v2v'],
+        required_flags=["v2v"],
+        fixture_name="source_provider",
         scope="module"
-    )
+    ),
+    pytest.mark.usefixtures("v2v_provider_setup")
 ]
 
 
-def get_migrated_vm_obj(src_vm_obj, target_provider):
-    """Returns the migrated_vm obj from target_provider."""
-    collection = target_provider.appliance.provider_based_collection(target_provider)
-    migrated_vm = collection.instantiate(src_vm_obj.name, target_provider)
-    return migrated_vm
-
-
-@pytest.mark.parametrize(
-    "form_data_vm_obj_single_datastore", [["nfs", "nfs", rhel7_minimal]], indirect=True
-)
-def test_migration_policy_tag(request, appliance, v2v_providers, host_creds, conversion_tags,
-                              form_data_vm_obj_single_datastore, soft_assert):
-    """Test policy to prevent source VM from starting if migration is complete
+@pytest.mark.tier(1)
+def test_migration_post_attribute(appliance, provider, mapping_data_vm_obj_mini, soft_assert):
+    """
+    Test to validate v2v post-migrations usecases
 
     Polarion:
         assignee: ytale
         initialEstimate: 1/4h
+        caseimportance: critical
+        caseposneg: positive
+        testtype: functional
+        startsin: 5.10
         casecomponent: V2V
```

Comment:
Looks good , please add the test steps too.

Reply:
Looks like I touched automation from long time, I don't remember I wrote teststeps before :D 
Adding it anyways, but do we still need it when we have added comments from which teststeps are fairly predictable, something like this https://github.com/ManageIQ/integration_tests/pull/8690/files/6c17c67b51b29753011eaa1dbc2aaffcced97820#diff-3e10df3dc335f2d18ff441a6bc8cba64R65 ?

Reply:
SO just add a summary of things that you are testing post migration in doc block , I see lot of asserts in test .

Reply:
Yup. Added it, thanks :)